rmd/30_BatchEffects.Rmd

```diff
@@ -175,24 +180,29 @@ In this paragraph, we're going to run the three functions key to Seurat Integrat
 # select features that are repeatedly variable across datasets for integration
 features <- SelectIntegrationFeatures(object.list = panc.list)
 
-anchors <- FindIntegrationAnchors(object.list = panc.list, anchor.features = features)
+anchors <- FindIntegrationAnchors(object.list = panc.list, anchor.features = features, normalization.method = "LogNormalize")
 
 # this command creates an 'integrated' data assay
-panc.combined <- IntegrateData(anchorset = anchors)
-
+panc.combined <- IntegrateData(anchorset = anchors, normalization.method = "LogNormalize")
 ```
 
 Let's clean up some redundant objects from memory.
 
 ```{r cleanup}
-rm(list=c("panc.list","features","anchors"))
+rm(list = c("panc.list", "features", "anchors"))
 gc()
 ```
 
 Let's have a brief look at the panc.combined dataset - a new Assay has been created by the integration procedure.
 
 
-**Poll** [What is the new assay called?](https://PollEv.com/multiple_choice_polls/4ArWqQjwtyVafBYCiaqbr/respond) Hint: you can access assays of a Seurat object with `Assays()`.
+> 🧭✨ Poll: 
+>
+> What is the new assay called?](https://PollEv.com/multiple_choice_polls/4ArWqQjwtyVafBYCiaqbr/respond) Hint: you can access assays of a Seurat object with `Assays()`.
+>
+> Which assay would be used if we run PCA now?
```

Comment:
I want to keep the explicit setting of the default assay, and explain why we are doing it and scaling data. I don't want to query students on this.